common/bloodlines/wc_bloodlines.txt

```diff
@@ -389,4 +389,18 @@ ymiron = {
 	allow_bastards = yes
```

Comment:
Over demesne limit:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/85869726-2125c400-b7dd-11ea-8fa3-9d4ae1ba4792.png)

</details>